docs/troubleshooting.md

```diff
@@ -24,6 +25,14 @@ dbug WebDriverAgent Device: Jul 26 13:21:57 iamPhone XCTRunner[240] <Warning>: E
 dbug WebDriverAgent Device: Jul 26 13:22:57 iamPhone XCTRunner[240] <Warning>: Enqueue Failure: UI Testing Failure - App state of (null) is still unknown <unknown> 0 1
 ```
 
+### Get a response after 60+ seconds after a session starts frequently
+
+Did you experience an additional 60 seconds of slow command response that is usually should not take such a long time?
+
+`testmanagerd` could continue crashing against a target application. Then, WebDriverAgent gets an API result by XCTest framework after 60 seconds timeout. It could cause such a 60 seconds "hang" behavior.
```

Comment:
It might be that the `testmanagerd` process on the device under test has crashed. In such case the OS tries to restore it causing the above delay while waiting for the resurrected daemon is connecting to the target process.